recOrder/pipelines/qlipp_pipeline.py

```diff
@@ -105,11 +106,14 @@ def __init__(self, config: ConfigReader, data: WaveorderReader, writer: Waveorde
                                                           gpu_id=self.config.gpu_id)
 
         # Compute BG stokes if necessary
-        if self.config.background_correction != 'None':
+        if self.config.background_correction == 'global':
+            print("Loading bg from "+self.bg_path)
             bg_data = load_bg(self.bg_path, self.img_dim[0], self.img_dim[1], self.bg_roi)
             self.bg_stokes = self.reconstructor.Stokes_recon(bg_data)
             self.bg_stokes = self.reconstructor.Stokes_transform(self.bg_stokes)
-
+        elif self.config.background_correction == 'local_fit':
+            self.bg_stokes = np.zeros((5, self.img_dim[0], self.img_dim[1]))
+            self.bg_stokes[0, ...] = 1 # Set background to "identity" Stokes parameters. waveorder's 'local_fit' applies this identity correction.
         else:
             self.bg_stokes = None
 
```

Comment:
Thanks for debugging this @talonchandler!

In the most common mode of reconstruction, we first apply the measured background (aka 'global' correction) and if needed we further fit and subtract a surface from this once already background corrected data. This is what I call "local" background correction, but we've seen that these names are poor. In practice it's a combination of "global" correction plus local fit.

The "global" background correction should simply subtract the background images, as is implemented here.

In this implementation, "local" background correction only subtracts the fitted 2d surface. I think it's better to rename this method and apply what I described at the top.

There are cases where we cannon acquire a background image and perform "global" background correction. In this case, we would only do surface fitting. You've implemented this here as providing unity background images.

So I think we need a total of four background correction methods:
* "none" - no background correction
* "measured" - use only background images
* "measured+" - use background images and surface fit to remove any residual background
* "estimated" - only use surface fit

Reply:
Aha! This context really helps me see why the waveorder code looks the way it does. I 100% agree with needing four options (either through a single four-option menu, two two-option menus, or two checkboxes). 

I was originally thinking of putting the "renaming of options" into it's own PR, but with this comment I think now I'll do my renaming here.